packages/@d-zero/git-diff/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import { execSync } from 'node:child_process';
+
+const tag =
+	process.argv[2] || execSync('git describe --tags --abbrev=0').toString('utf8').trim();
+const tag2 = process.argv[3] || 'HEAD';
```

Comment:
```suggestion
const tag2 = process.argv[3] ?? 'HEAD';
```

@yusasa16 

論理値ではないものの代入なので `||` ではなく `??` を使いましょう。